### ISSUE TITLE
Build fails on autoconf 1.69

### ISSUE THREAD

**Issue by BrettRD** [CIFASIS/modelicacc#69]
The build configuration does not copy two files from the system, and won't build without them
```
./configure
configure: error: cannot find required auxiliary files: config.guess config.sub
```
I'm building on a fresh docker container at ubuntu:22.04 on x64, with only the specified dependencies (except boost 1.74 not 1.71)

I dug up some history that I don't understand and called `autoreconf -I m4 --install `, it generates the makefiles after that.
It left me with a remaining warning:
`autoreconf: warning: autoconf input should be named 'configure.ac', not 'configure.in'`

**Comment by BrettRD** [comment]
This also bug appears in the sbg dependency, and can be resolved with the same command

The boost dependency in the readme also needs bumping, the sbg dependency requires unordered_flat_set.hpp which is not available in boost 1.74

**Comment by joaquinffernandez** [comment]
Hi @BrettRDthanks for reporting this issue. I'll the deps for U22 builds in both repos.
